README.rst

```diff
@@ -1627,6 +1672,19 @@ The signal is emitted when the device name changes.
 
 It is not emitted when the device is created.
 
+``device_group_changed``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Path**: ``openwisp_controller.config.signals.device_group_changed``
+
+**Arguments**:
+
+- ``instance``: instance of ``Device``.
+
+The signal is emitted when the device group changes.
+
+It is not emitted when the device is created.
```

Comment:
would be good to have old group and new group, we do the same in other similar signals in this module and others, eg: https://github.com/openwisp/openwisp-controller#management-ip-changed